templates/core/core-cm.yaml

```diff
@@ -36,6 +36,7 @@ data:
   CLAIR_DB: "{{ template "harbor.database.clairDatabase" . }}"
   CLAIR_DB_SSLMODE: "{{ template "harbor.database.sslmode" . }}"
   CLAIR_URL: "http://{{ template "harbor.fullname" . }}-clair:6060"
+  CLAIR_ADAPTER_URL: "http://{{ template "harbor.clair" . }}:8080"
```

Comment:
We should only need clair_adapter_url in core's CM?

Reply:
We'll remove this variables after they are deleted from main repository